src/code42cli/logger/handlers.py

```diff
@@ -0,0 +1,122 @@
+import logging
+import socket
+import ssl
+from logging.handlers import SysLogHandler
+
+from code42cli.cmds.search.enums import ServerProtocol
+
+
+class NoPrioritySysLogHandler(SysLogHandler):
```

Comment:
THIS! This is the best syslog server logger handler in python on the internet haha. Well, that is the intent anyway.

It combines the current state of the syslog handler code with the socket wrapping logic I learned from https://github.com/lhl/python-syslogssl

PLUS it still strips out the Priority related logic that we were not using